ursa/graph/utils.py

```diff
@@ -44,3 +47,64 @@ def _apply_append(collection, values):
             collection.update(val)
 
         return collection
+
+
+@ray.remote(num_return_vals=0)
+def write_vertex(vertex, graph_id, key):
```

Comment:
This needs proper docstring commenting (see Slack message for reference)

Reply:
Resolved.